src/js/controllers/imageCtrl-test.js

```diff
@@ -0,0 +1,142 @@
+'use strict';
+
+describe('imagesCtrl', function() {
+    var $rootScope, $controller, imagesCtrl, images, vm, $q, mockImageObject, $log;
+
+    mockImageObject = {
+        contentType: "text/json",
+        status: "success",
+        photos: {
+            photo: [
+                {
+                    farm: 6,
+                    id: "23780914905",
+                    isfamily: 0,
+                    isfriend: 0,
+                    ispublic: 1,
+                    owner: "138698049@N03",
+                    secret: "0eb2a77a55",
+                    server: "5764",
+                    title: ""
+                },
+                {
+                    farm: 1,
+                    id: "23698470321",
+                    isfamily: 0,
+                    isfriend: 0,
+                    ispublic: 1,
+                    owner: "138698049@N03",
+                    secret: "1d4dc255d0",
+                    server: "693",
+                    title: ""
+                }
+            ]
+        }
+    };
+
+    beforeEach(module('flickrPOC'));
+
+    describe('imagesCtrl', function(){
+
+        describe('Sucessful call to api', function(){
+            beforeEach(module(function($provide){
+                $provide.service('images', function($q){
+                    this.loadPics = sinon.stub().returns($q.resolve(mockImageObject));
+                });
+
+            }));
+            beforeEach(inject(function (_$rootScope_, _$controller_, _images_, _$q_, _$log_) {
+                $rootScope = _$rootScope_.$new();
+                $controller = _$controller_;
+                images = _images_;
+                $q = _$q_;
+                $log = _$log_;
+                vm = $controller('imagesCtrl');
+
+                images.loadPics.returns($q.resolve(mockImageObject));
```

Comment:
This isn't needed since you're initializing the return value in the $provide block.